scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -473,6 +478,13 @@
                       {:max-synthetic-pods max-pods-outstanding
                        :synthetic-pods num-synthetic-pods}))
 
+          (set-counter-fn "total-pods" total-pods)
+          (set-counter-fn "max-total-pods" max-total-pods)
+          (set-counter-fn "total-nodes" total-nodes)
+          (set-counter-fn "max-total-nodes" max-total-nodes)
+          (set-counter-fn "total-synthetic-pods" num-synthetic-pods)
+          (set-counter-fn "max-total-synthetic-pods" max-pods-outstanding)
```

Comment:
I just realized that these metrics are being set inside `autoscale!`, which means that pools that don't autoscale won't have these metrics. That's fine for the synthetic pods metrics and for `max-total-pods` and `max-total-nodes`, but for `total-pods` and (maybe) `total-nodes`, it might be nice to have these metrics even for non-autoscaling pools. Thoughts?

Reply:
yeah, let me see if there's a better place to put this